generator/whitelist.ts

```diff
@@ -4,6 +4,16 @@ import { lowerCaseEquals } from './utils';
 
 // Run "npm run list-basepaths" to discover all the valid readme files to add to this list
 const whitelist: WhitelistConfig[] = [
+    {
+        basePath: 'advisor/resource-manager',
+        namespace: 'Microsoft.Advisor',
+        resourceConfig: [
+            {
+                type: 'recommendations/suppressions',
+                scopes: ScopeType.Tenant | ScopeType.Subcription | ScopeType.ResourceGroup | ScopeType.ManagementGroup,
```

Comment:
Is this correct? From the swagger, doesn't look like this resource is deployable at tenant, subscription or Rg scopes.

Reply:
The path in [2020-01-01/advisor.json#L505](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json#L505) is `"/{resourceUri}/providers/Microsoft.Advisor/recommendations/{recommendationId}/suppressions/{name}":` 
Can I ask how to tell if it's deployable at a scope or not?

Reply:
There's no easy way to figure this out. Looking at the docs [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/extension-resource-types) and [here](https://docs.microsoft.com/en-us/rest/api/advisor/suppressions/create) it does appear this is an extension resource. Please could you mark this as `ScopeType.Extension` and re-run the autogeneration?

Reply:
@leni-msft any updates on this?

Reply:
resolved. pls check again.